sci-electronics/magic/magic-8.3.309.ebuild

```diff
@@ -69,6 +72,7 @@ src_configure() {
 		--disable-modular \
 		$(use_enable debug memdebug) \
 		$(use_enable cairo cairo-offscreen) \
+		$(use_with cairo) \
```

Comment:
I think this is going to require a revbump. The installed files will be different after this change for user who have USE="cairo".